arrays/arrays.c

```diff
@@ -21,11 +21,15 @@ typedef struct Array {
  *****/
 Array *create_array (int capacity) {
   // Allocate memory for the Array struct
-
+  Array *array = mallac(sizeof(Array));
   // Set initial values for capacity and count
-
+  array->capacity = capacity;
+  array->count = 0;
   // Allocate memory for elements
+  char **elements = malloc(capacity * sizeof(char));
```

Comment:
`sizeof(char *)` - remember, this is an array storing _pointers_ (pointers that point to chars), it's not holding chars.